lib/tasks/add_browse_users.rake

```diff
@@ -0,0 +1,42 @@
+# rake organisation:add_browse_users
+namespace :organisation do
+  task add_browse_users: :environment do
```

Comment:
Provide some example usage in the description. If the environment variables are missing, return a message with the usage information.

Could you also change the namespace and task name to `goodcity:add_user` - we'll use it for more than just browse.